packages/app/src/cli/services/dev.ts

```diff
@@ -108,6 +109,7 @@ async function dev(options: DevOptions) {
   if (options.app.extensions.ui.length > 0) {
     const devExt = await devUIExtensionsTarget({
       app: options.app,
+      id: app.id,
```

Comment:
@isaacroldan we are passing it off the app object here, there was a typo before where it was `options.id` but we have caught that now